src/uwtools/drivers/driver.py

```diff
@@ -154,9 +168,9 @@ def _create_user_updated_config(
 
     # Public helper methods
 
-    @property
+    @classmethod
```

Comment:
Sadly, use of `@classmethod` and `@property` together [has been deprecated](https://docs.python.org/3.11/library/functions.html#classmethod) in Python.